skaffold.yaml

```diff
@@ -7,5 +7,12 @@ build:
   artifacts: []
 
 deploy:
-  kubectl:
-    remoteManifests: []
+  helm:
+    releases:
+      - name: dashy
+        namespace: dashy
+        createNamespace: true
+        chartPath: charts/dashy
+        valuesFiles:
+        - values/dashy/values.yaml
```

Comment:
Here you can put whatever overrides you want for your ingress.

Mine looks like this;


# Leave empty if you plan to use config file from host path

ingress:
  enabled: true
  className: "traefik"
  annotations: {}
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
  hosts:
    - host: dashy.lakshmi.lan
      paths:
        - path: /
          pathType: ImplementationSpecific
  tls: []

#     pathType: Prefix
#     backend:
#      service:
#        name: exampleService
#        port: 80

dashyConfig:
  data: |
    appConfig:
      theme: matrix